tests/foreman/api/test_contentview.py

```diff
@@ -185,6 +186,31 @@ def test_positive_add_custom_content(self):
         self.assertEqual(len(content_view.repository), 1)
         self.assertEqual(content_view.repository[0].read().name, yum_repo.name)
 
+    @tier2
+    @run_only_on('sat')
+    def test_positive_add_custom_module_streams(self):
+        """Associate custom content (module streams) in a view
+
+        :id: 9e4821cb-293a-4d84-bd1f-bb9fff36b143
+
+        :expectedresults: Custom content (module streams) assigned and
+        present in content view
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        yum_repo = entities.Repository(product=product,
+                                       url=CUSTOM_MODULE_STREAM_REPO_2).create()
+        yum_repo.sync()
+        content_view = entities.ContentView(organization=org).create()
+        self.assertEqual(len(content_view.repository), 0)
+        content_view.repository = [yum_repo]
+        content_view = content_view.update(['repository'])
+        self.assertEqual(len(content_view.repository), 1)
+        self.assertEqual(content_view.repository[0].read().name, yum_repo.name)
+        self.assertGreater(content_view.repository[0].read().content_counts['module_stream'], 5)
```

Comment:
I can't help but notice that you use `assertGreater` and 5 here, but `assertGreaterEqual` and 5 in another case, against the same repository.

It doesn't really matter, but you might want to make these assertions consistent. Feel free to resolve this conversation without any code change.

Reply:
Yeah make sense ! will update that.